extensions-contrib/opencensus-extensions/src/test/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParserTest.java

```diff
@@ -139,6 +157,70 @@ public void testSummaryParse() throws Exception
 
   }
 
+  @Test
+  public void testDimensionsParseWithParseSpecDimensions() throws Exception
+  {
+    //configure parser with desc file
+    OpenCensusProtobufInputRowParser parser = new OpenCensusProtobufInputRowParser(parseSpecWithDimensions);
+
+    DateTime dateTime = new DateTime(2019, 07, 12, 9, 30, ISOChronology.getInstanceUTC());
+
+    Timestamp timestamp = Timestamp.newBuilder().setSeconds(dateTime.getMillis() / 1000)
+        .setNanos((int) ((dateTime.getMillis() % 1000) * 1000000)).build();
+
+    Metric d = summaryMetric(timestamp);
```

Comment:
Can you use a more descriptive name than `d`?

Reply:
My bad, just followed previous test cases. Corrected in all test cases.